internal/repositories/accountMongoRepository.go

```diff
@@ -0,0 +1,149 @@
+package repositories
```

Comment:
Since this is just an implementation, it does not make sense to live in this package. 
For example, k8s uses pattern of creating, let's say `/internal/repositories/interface.go` where you would define interfaces.
Then, your implementation of the repo would go to `/internal/repositories/mongo` for example.